src/recipes/editor-plugin.md

```diff
@@ -0,0 +1,56 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Editor plugins
+
+
+Using `EditorPlugin` types is very similar to the process used when [writing plugins in GDScript][gd-plugins].
+Unlike GDScript plugins, godot-rust plugins are registered automatically and cannot be enabled/disabled in the
+Project Settings plugins pane.
+
+<!-- This section may be unecessary? -->
+Plugins written in GDScript are automatically disabled if they have a code error, but because Rust is a compiled language,
+you cannot introduce compile-time errors because the library binary will simply not be emitted on the compiler finding an error.
+This is not the case for run-time errors.
```

Comment:
I think the section is fine, but the double "because" makes the sentence hard. Just omit the 2nd part; it goes probably too much into detail or repeats itself.
```suggestion
Plugins written in GDScript are automatically disabled if they have a code error, but because Rust is a compiled language,
you cannot introduce compile-time errors.
```